package.json

```diff
@@ -10,15 +10,13 @@
   "type": "module",
   "exports": {
     ".": {
-      "browser": {
```

Comment:
browser was moved under `import` to reduce the complexity and repetition of having `types` under 3 separate entries.